themes/jaeger-docs/layouts/docs/single.html

```diff
@@ -14,6 +14,7 @@
 
       <div class="column is-8 is-full-touch">
         <section class="content content--docs is-medium">
+          <p class="title is-1">Single.html</p>
```

Comment:
is this a debugging left-over?

Reply:
@celestehorgan should this line be removed?

Reply:
Yes – sorry for letting this go stale, it's been a busy few weeks :)